From: sorah <sorah@b2dd03c8-39d4-4d8f-98ff-823fe69b080e>
Date: Sun, 30 Dec 2018 03:15:58 +0000
Subject: Unset SOURCE_DATE_EPOCH on Rubygems test cases

Relates to r66554. When test suites run with SOURCE_DATE_EPOCH, several
tests would fail.  Environment Variables are already kept with this
setup & teardown phase.

git-svn-id: svn+ssh://ci.ruby-lang.org/ruby/trunk@66648 b2dd03c8-39d4-4d8f-98ff-823fe69b080e
---
 lib/rubygems/test_case.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/lib/rubygems/test_case.rb b/lib/rubygems/test_case.rb
index 7648ffd..a93f749 100644
--- a/lib/rubygems/test_case.rb
+++ b/lib/rubygems/test_case.rb
@@ -254,6 +254,7 @@ def setup
     @orig_gem_env_requirements = ENV.to_hash
 
     ENV['GEM_VENDOR'] = nil
+    ENV['SOURCE_DATE_EPOCH'] = nil
 
     @current_dir = Dir.pwd
     @fetcher     = nil
